docker/Dockerfile.ci_gpu

```diff
@@ -83,6 +83,9 @@ RUN bash /install/ubuntu_install_dgl.sh
 COPY install/ubuntu_install_vulkan.sh /install/ubuntu_install_vulkan.sh
 RUN bash /install/ubuntu_install_vulkan.sh
 
+COPY install/ubuntu_install_caffe.sh /install/ubuntu_install_caffe.sh
```

Comment:
Can we just install the caffe on the CPU and run ci tests through the CPU side? This would reduce our pressure in terms of depending on the GPU resources

Reply:
We can add the tests to here https://github.com/apache/incubator-tvm/blob/master/tests/scripts/task_python_frontend_cpu.sh

Reply:
Good, if we just install the caffe on the CPU, it will reduce a lot of work.